bitnami/contour/templates/contour/deployment.yaml

```diff
@@ -67,6 +67,9 @@ spec:
             {{- if .Values.ingressClass }}
             - --ingress-class-name={{ .Values.ingressClass }}
             {{- end }}
+            {{- if .Values.contour.extraArgs }}
+            {{ toYaml .Values.contour.extraArgs | indent 12 | trim }}
```

Comment:
Maybe it could be even better using `common.tplvalues.render` helper, don't you think?

Reply:
Sounds good! Thanks for the feedback.